test/suite/test_cursor_bound01.py

```diff
@@ -41,6 +41,7 @@ class test_cursor_bound01(bound_base):
         ('lsm', dict(uri='lsm:', use_index = False, use_colgroup = False)),
         ('colgroup', dict(uri='table:', use_index = False, use_colgroup = False)),
         ('index', dict(uri='table:', use_index = True, use_colgroup = False)),
+#        ('layered', dict(uri='layered:', use_index = False, use_colgroup = False)),
```

Comment:
What's the deal with this? Is it something we'd expect to turn on before merging?

Reply:
I could not turn this on until SLS-989 was merged, since it was causing test fallout for me. I'll try again.